test-gui.sh

```diff
@@ -3,6 +3,38 @@
 set -exuo pipefail
 
 cd frontend
+
+if [ -v FORCE_CHRMOE_BRANCH_BASE ];
+then
+  BRANCH_BASE=${FORCE_CHRMOE_BRANCH_BASE}
+  export CHROME_BINARY_PATH="${PWD}/__chrome_browser__/${BRANCH_BASE}/chrome-linux/chrome"
+
+  # look for chrome binary
+  if [ -x "${CHROME_BINARY_PATH}" ];
+  then
+    echo "chrmoe binary for branch ${BRANCH_BASE} already exists"
+  else
+    CHROME_DIR="${PWD}/__chrome_browser__"
+    CHROME_DOWNLOAD_URL="https://www.googleapis.com/download/storage/v1/b/chromium-browser-snapshots"
+
+    # download
+    mkdir -p "${CHROME_DIR}/${BRANCH_BASE}"
+    curl -G "${CHROME_DOWNLOAD_URL}/o/Linux_x64%2F${BRANCH_BASE}%2Fchrome-linux.zip" \
+        -d "alt=media" \
+        > "${CHROME_DIR}/chrome-linux-${BRANCH_BASE}.zip"
+    unzip "${CHROME_DIR}/chrome-linux-${BRANCH_BASE}.zip" -d "${CHROME_DIR}/${BRANCH_BASE}"
```

Comment:
nitpick: you can use pipe instead of saving the zip to disk, the internet will thank you :-)

Reply:
@ohadlevy 
the unzip on my system does not accept pipe streams :-(
I tried using `tar` and `funzip` but they also failed to make them work here :disappointed: 
what am I missing ?